### PR TITLE
fix: add emission control

### PR DESCRIPTION
**Pull request by axejintao**
# Summary 

- add emission control support to sdk

### CODE REVIEW

File: src/rewards/rewards.service.ts [hunk 1]
```diff
@@ -178,6 +203,25 @@ export class RewardsService extends Service {
...
+      // TODO: enable once emission control is on the registry
+      // if (emissionControlAddress) {
+      //   this._emissionControl = EmisisonControl__factory.connect(
+      //     emissionControlAddress,
+      //     this.connector,
+      //   );
+      // }
+      if (this.config.network === Network.Ethereum) {
+        this._emissionControl = EmisisonControl__factory.connect(
+          '0x31825c0a6278b89338970e3eb979b05b27faa263',
```
Comment (mrbasado) [inline src/rewards/rewards.service.ts:215]: can these addresses be somewhere else

**Review by mrbasado — APPROVED** [review]

File: src/rewards/rewards.service.ts (hunk 1 above)
Comment (axejintao) [reply src/rewards/rewards.service.ts:215]: resolved offline, will be removed once the emission control is on the registry, will submit pr tmrw

**Pull request merged**